metadata-suggestions.md

```diff
@@ -0,0 +1,528 @@
+        "_links": {
+          "entries": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/orcid/entries"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/orcid"
+          }
+        }
+      },
+      {
+        "id": "ciencia",
+        "name": "ciencia",
+        "type": "metadataSuggestion",
+        "query-based": "true",
+        "file-based": "false",
+        "metadata-based": "true",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/ciencia/entries"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/ciencia"
+          }
+        }
+      },
+      {
+        "id": "ris_data_loader",
+        "name": "ris_data_loader",
+        "type": "metadataSuggestion",
+        "query-based": "false",
+        "file-based": "true",
+        "metadata-based": "false",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/ris_data_loader/entries"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/ris_data_loader"
+          }
+        }
+      }
+    ]
+  },
+  "_links": {
+    "self": {
+      "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions"
+    }
+  },
+  "page": {
+    "size": 20,
+    "totalElements": 3,
+    "totalPages": 1,
+    "number": 0
+  }
+}
+```
+
+## Single suggestion endpoint
+**/api/integration/metadata-suggestions/<:suggestion-name>**
+
+Provide detailed information about a specific external source. The JSON response document is as follow
+```json
+{
+    "id": "pubmed",
+    "name": "pubmed",
+    "type": "metadataSuggestion",
+    "query-based": "true",
+    "file-based": "false",
+    "metadata-based": "true",
+    "_links": {
+      "entries": {
+        "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/pubmed/entries"
+      },
+      "self": {
+        "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/pubmed"
+      }
+    }
+}
+```
+
+Properties:
+* query-based: if the external source uses a query to suggest information to import
+* file-based: if the external source uses a file to suggest information to import
+* metadata-based: if the external source uses the current item metadata to suggest information to import
+
+Exposed links:
+* entries: the list of values managed by the external source
+
+## Linked entities
+### External source entries
+**/api/integration/metadata-suggestions/<:suggestion-name>/entries**
+**/api/submission/workflowitems/<:id>/metadata-suggestions/<:suggestion-name>/entries**
```

Comment:
I'm not sure why this second path exists. Is this a mistake?  It seems like this endpoint should only exist under one path, not two.

Reply:
This was indeed a mistake, I've updated it now